CHANGELOG.md

```diff
@@ -77,6 +77,7 @@
   instance for consistency with other OneOf field accessors (#5035)
 - Update OTLP to v0.15.0 (#5064)
 - Adding support for transition from older versions of OTLP to OTLP v0.15.0 (#5085)
+- Split pdata package separated by telemetry signal type (#5087)
```

Comment:
Please add a note about the deprecation of `pdata.*` aliases as well

Reply:
Replaced by an entry in "Deprecations" section